### DISCUSSION TITLE
Pool mining : explanation and can we use old plots to avoid to renew all (above all if 1000 plots or more)

### DISCUSSION THREAD

**Discussion by casa10** [Chia-Network/chia-blockchain#6022] [General]
hye,
About pool mining there are these informations : 

**Will I need to replot to use the official pooling protocol?**
Yes. Anyone who wants to join a pool will need to create new K32 or above portable plots. This new plot format allows you to switch between pools and self-pooling with a cooldown of 30 minutes (rules set by pool) between each switch. Each switch between pools will require a transaction with a smart contract on the blockchain. Our recommendation is to slowly replace your existing plots with portable plots one by one, so you still have a chance to win XCH while you convert to all portable plots.

**Can I farm with both OG (original) plots and portable plots?**
Yes. The farmer will support both OG plots and portable plots on one machine.

**My question**
1) can we keep our current plots to continue to farm in solo and in same time create the new plots (portable or the new k32) for only participate to a pool ? (a part of your space to farm in solo with "old" way to make plots and another part of your space for pooling (or self-pooling) The goal is to avoid to destroy the old plots, above all if you had already 500,1000 plots ! or more.

**Comment by tiny0little** [comment]
yes, i think you can do that -> slow transition from solo to official pooling
